parts/kubernetesmastercustomscript.sh

```diff
@@ -95,6 +95,12 @@ function ensureKubectl() {
     fi
 }
 
+function downloadUrl () {
```

Comment:
This is a great improvement.  Please adapt all places in this script to use download URL (I looked and there is only one other).